### PR TITLE
Личный проект: подробности

### PR DESCRIPTION
**Pull request by heyiamvika**
---
:mortar_board: [Личный проект: подробности](https://up.htmlacademy.ru/javascript/13/user/474591/tasks/15)

:boom: https://htmlacademy-javascript.github.io/474591-keksobooking/3/

### CODE REVIEW

File: js/map.js [hunk 1]
```diff
@@ -90,28 +95,43 @@ var generateSimilarListings = function () {
 
 var createPins = function (pin) {
```
Comment (rkhaslarov) [inline js/map.js]: renderPin

File: js/map.js [hunk 2]
```diff
@@ -90,28 +95,43 @@ var generateSimilarListings = function () {
...
+        var card = generateCard(listings[pins[x].dataset.index])
+      })
+    }
+
+    createEventListener(i)
+  }
 }
 
 var generateCard = function (place) {
+  console.log('place: ', place)
```
Comment (rkhaslarov) [inline js/map.js]: Should be removed

File: js/map.js [hunk 3]
```diff
@@ -140,11 +160,39 @@ var generateCard = function (place) {
...
+var activateForm = function () {
+  map.classList.remove('map--faded')
+  adForm.classList.remove('ad-form--disabled')
+  adFormFieldset.forEach(function (element) {
+    element.removeAttribute('disabled')
+  })
+}
+
+var defineAddressUnactivated = function () {
+  var xInitial = parseInt(mainPin.style.left, 10)
```
Comment (rkhaslarov) [inline js/map.js]: Better to return value and then assign it to fieldsetAddress.value

File: js/map.js [hunk 4]
```diff
@@ -140,11 +160,39 @@ var generateCard = function (place) {
...
+  var yInitial = parseInt(mainPin.style.top, 10)
+  fieldsetAddress.value = xInitial + ', ' + yInitial
+}
+
+var defineAddressActivated = function () {
+  var xAfterDragged = parseInt(pinTemplate.style.left, 10) + pinImage.width / 2
```
Comment (rkhaslarov) [inline js/map.js]: The same as above

File: js/map.js (hunk 2 above)
Comment (heyiamvika) [reply js/map.js]: done

File: js/map.js (hunk 3 above)
Comment (heyiamvika) [reply js/map.js]: done

File: js/map.js (hunk 1 above)
Comment (heyiamvika) [reply js/map.js]: done

File: js/map.js (hunk 4 above)
Comment (heyiamvika) [reply js/map.js]: done

**Pull request merged**